magstock/templates/regextra.html

```diff
@@ -19,7 +19,7 @@
         setVisible('#allergies', $.field('purchased_food').is(':checked'));
     };
     var showOrHideComingWith = function () {
-        var showComingWith = !!$.val('coming_as');
+        var showComingWith = ($.val('coming_as') === {{ c.TENT_LEADER }} || $.val('coming_as') === {{ c.TENT_FOLLOWER }});
```

Comment:
Minor style point: the parenthesis aren't necessary here.  It's generally accepted in Javascript to say

```javascript
var foo = bar || baz;
```

without the use of parenthesis.  Either way is fine though, so I'm going to go ahead and just merge this in.